src/net_processing.h

```diff
@@ -73,6 +73,14 @@ class PeerLogicValidation final : public CValidationInterface, public NetEventsI
     */
     bool SendMessages(CNode* pto) override EXCLUSIVE_LOCKS_REQUIRED(pto->cs_sendProcessing);
 
+    /**
+    * Process global tasks that aren't attached to a specific peer
```

Comment:
Wondering if we should clarify that these tasks are expected to be limited to 1 second or something (perhaps "lightweight tasks")?
Just so that future devs don't tempt to put any heavy logic in here.

Reply:
Such documentation, if it's added, should be at the level of the `PeerLogicValidation` class. We don't want anything in the message handler thread to block for a long time (or to *be blocked* by slow tasks in other threads). That includes anything in  `ProcessMessages()`, `SendMessages()` and (now) `ProcessGlobalTasks()`.